src/common/components/repository-input/index.js

```diff
@@ -48,6 +50,24 @@ export class RepositoryInput extends Component {
     const isTouched = input.inputValue.length > 2;
     const isValid = !!input.repository && !input.error;
 
+    let submitButton;
+    if (conf.get('LP_API_USERNAME')) {
+      // Main path, for use in production.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Creating...' : 'Create' }
+        </Button>
+      );
+    } else {
+      // If the Launchpad API isn't configured, fall back to just verifying
+      // the repository.  This is handy in development.
+      submitButton = (
+        <Button type='submit' disabled={!isValid || input.isFetching}>
+          { input.isFetching ? 'Verifying...' : 'Verify' }
+        </Button>
+      );
+    }
```

Comment:
I've split out a separate `submitButton` method.

Yes, I agree this overlaps with your PR, and I don't mean this to supersede yours; I just wanted to get to the point of being able to actually test that we have working end-to-end snap creation as quickly as possible.  I don't mind if you then replace it with your flow.  (Note though that we don't yet have a backend method to dispatch a build ...)